Package.swift

```diff
@@ -7,15 +7,17 @@ let package = Package(
     name: "bindings-swift",
     platforms: [
         .macOS(.v12),
-        .iOS(.v11),
+        .iOS(.v14),
     ],
     products: [
-        .library(name: "BreezSDK", targets: ["breez_sdkFFI", "BreezSDK"]),
+        .library(name: "BreezSDK", targets: ["breez_sdkFFI", "BreezSDK", "BreezSDKNotification"]),
```

Comment:
Is the `BreezSDKNotification` target generated automatically or do we need to declare it explicityl below in the targets array? 🤔

Reply:
Good point, I think I'm thinking about it backwards. I should have 3 targets and 1 library:
```
targets: [
        .binaryTarget(name: "breez_sdkFFI", url: "https://github.com/breez/breez-sdk-swift/releases/download/0.3.1/breez_sdkFFI.xcframework.zip", checksum: "fcda9a87b512fc0ab81619096d24db6e54b622c18cded6d1d78db95f05c10265"),
        .target(name: "BreezSDK", dependencies: ["breez_sdkFFI"]),
        .target(name: "BreezSDKNotification", dependencies: ["BreezSDK", "KeychainAccess", "XCGLogger"]),
    ]
```

Reply:
Does this require a change on 
- https://github.com/breez/breez-sdk/pull/834

?

Reply:
No, it should pickup the target source files by directory name "Sources/BreezSDKNotification"